checking for libxslt ... not found
checking for libxslt in /usr/local/ ... not found
checking for libxslt in /usr/pkg/ ... not found
checking for libxslt in /opt/local/ ... not found

./configure: error: the HTTP XSLT module requires the libxml2/libxslt
libraries. You can either do not enable the module or install the libraries.

---

--- a/auto/lib/libxslt/conf
+++ b/auto/lib/libxslt/conf
@@ -12,7 +12,7 @@
                       #include <libxslt/xsltInternals.h>
                       #include <libxslt/transform.h>
                       #include <libxslt/xsltutils.h>"
-    ngx_feature_path="/usr/include/libxml2"
+    ngx_feature_path="${STAGING_DIR}/opt/include/libxml2"
     ngx_feature_libs="-lxml2 -lxslt"
     ngx_feature_test="xmlParserCtxtPtr    ctxt = NULL;
                       xsltStylesheetPtr   sheet = NULL;
@@ -100,7 +100,7 @@ fi
     ngx_feature_name=NGX_HAVE_EXSLT
     ngx_feature_run=no
     ngx_feature_incs="#include <libexslt/exslt.h>"
-    ngx_feature_path="/usr/include/libxml2"
+    ngx_feature_path="${STAGING_DIR}/opt/include/libxml2"
     ngx_feature_libs="-lexslt"
     ngx_feature_test="exsltRegisterAll();"
     . auto/feature
